### PR TITLE
Use isparta for coverage is babel is chosen

### LINKED ISSUE #162: Should use isparta for coverage if babel is chosen
Istanbul only runs on the compiled JavaScript so it doesn't work with any preprocessor. https://github.com/douglasduteil/isparta should probably be used for babel.

### PR DESCRIPTION
**Pull request by stevemao**
Fixes #162

### CODE REVIEW

File: generators/gulp/templates/gulpfile.js [hunk 1]
```diff
@@ -34,10 +35,12 @@ gulp.task('nsp', function (cb) {
 
 gulp.task('pre-test', function () {
   return gulp.src('lib/**/*.js')
+  .pipe(istanbul({
+    includeUntested: true,
```
Comment (SBoudrias) [inline generators/gulp/templates/gulpfile.js]: The trailing comma shouldn't be there if we only have one line

**Comment by SBoudrias** [comment]
Did you actually test the feature?

I don't see how the code is going to be runned if it is not compiled first.

**Comment by stevemao** [comment]
The code can't be compiled because it will be normal javascript code in vinyl and instrumenter can't know the original code. I copied the code from you :)

```js
gulp.src('lib/**.js')
  .pipe(istanbul({
    instrumenter: isparta.Instrumenter
  }));
```

**Comment by SBoudrias** [comment]
Oh you're right, I was confused for a moment.

**Pull request merged**